### DISCUSSION TITLE
Kompass- Himmelsrichtung tot

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#387] [Q&A] [answered]
Seit dem letzten Update bin ich total happy! 
Ein kleines Problem habe ich noch, hat aber Priorität: Null, zumal die Saison vorbei ist. Ich benutze aber den Cruiser lieber, als mein Navigation im Auto.  Ich fahre immer bei der Navigation mit dem Kompass im 2D, selten 3D. Aber niemals in N-Richtungsanzeige. Früher genügte ein Tip auf den Kompass  um die gewünschte 2D- Anzeige in der Navigation zu bekommen. Gibt's einen Trick?
Herzlichst, Antonio

**Comment by kds112** [comment]
A long-press does not help? Normally, the menu should appear with long-press. A shorts tap has no function in the moment. There is another discussion to add a (user-defined) function...

**Reply by SF4User** [reply]
Nein, auch lange drauf drücken hilft nicht. Habe ich natürlich vor dem Post gemacht/probiert, immer wieder - weil ich mir schon gedacht habe, daß das mit dem letzten Update zusammenhängt. Nun, nach einiger Zeit, dachte ich, ich melde das mal . . .

**Comment by devemux86** [comment]
@SF4User 

**Long press** on the compass to select the map orientation mode.
(movement direction 2D or 3D or no rotation)

**Drücken Sie lange** auf den Kompass, um den Kartenausrichtungsmodus auszuwählen.
(Bewegungsrichtung 2D oder 3D oder keine Drehung)

**Reply by SF4User** [reply]
Länger Druck funktioniert nicht, egal ob Autohide eingeschaltet oder nicht. Habe alle  Möglichkeiten durchdekliniert. Ist auch erst seit dem letzten Update. Seitdem übrigens sehr oft Absturz der App Cruiser - was früher nie der Fall war.

**Reply by devemux86** [reply]
@SF4User 

There are no changes in the compass. If you install Cruiser 3.0.36 from Google Play, does it work?
About the crash, what exactly were you doing?

Es gibt keine Änderungen im Kompass. Wenn Sie Cruiser 3.0.36 von Google Play installieren, funktioniert es?
Über den Absturz, was genau haben Sie getan?

**Reply by devemux86** [reply]
@SF4User 

Are you using another app with Cruiser that overlays the screen?
Have you restarted the phone?

Verwenden Sie eine andere App mit Cruiser, die den Bildschirm überlagert?
Haben Sie das Telefon neu gestartet?

**Reply by SF4User** [reply]
Der Absturz der App passiert, wenn dann immer, wenn ich in die Einstellungen wechsle. Ob die Navigation dabei ein- oder ausgeschaltet ist, bin ich nicht sicher - das muss ich mal gezielt verfolgen. Andere Apps sind immer abgeschaltet. Das passierte jetzt immer wieder, als ich die Einstellung gesucht/geändert habe, um den Kompass umzustellen. Irgendwann wird sich das schon auflösen. Ich habe immer den aktuellen  Cruiser installiert, derzeit den 3.0.36

**Reply by SF4User** [reply]
Ich hatte eben drei App Abstürze kurz hintereinander provozieren können  . . . immer im Zusammenhang, wenn die Navigstion eingeschaltet war oder wenn ich die Navigation ein- und ausgeschaltet habe und Einstellungen verändert habe, z.B. den Bottom für die eigene Position oder den Zoom von der Karte.

**Reply by SF4User** [reply]
Einen Absturz der App kann ich immer erzeugen, wenn ich die Karte mit zwei Finger heraus zoome , meinen Standort weit weg schiebe . . . z.B. nach rechts oben und dann den Bottom rechts unten: Mein Standort drücke. Navigation ein / aus egal.

**Reply by devemux86** [reply]
@SF4User 

First backup your favorites and routes / tracks.
You can uninstall and install again Cruiser (or reset the current profile), to see if some profile setting is the problem.
If the problem continues, please post your profile and a video if possible to see what happens.

Sichern Sie zunächst Ihre Favoriten und Routen/Tracks.
Sie können Cruiser deinstallieren und erneut installieren (oder das aktuelle Profil zurücksetzen), um zu sehen, ob eine im Profil Einstellung das Problem ist.
Wenn das Problem weiterhin besteht, posten Sie bitte Ihr Profil und ein Video, wenn möglich, um zu sehen, was passiert.

**Reply by SF4User** [reply]
After resetting and uninstalling + reinstalling the Cruiser app, the crashes are resolved. But the app still has a little bug: VTM (OpenGL) must be set under the Settings/Map/Rendering Engine. The compass change does not work with Mapsforge Option -  only with VTM(OpenGL) Option. For me, all Problems adjusted.

**Reply by devemux86** [reply, marked as answer]
VTM (OpenGL) is the default map engine,

as Mapsforge does not support map rotation, so the map orientation (via compass button) is of no use.

**Comment by devemux86** [comment]
@SF4User 

If the compass is set to auto-hide, first press the display to show the compass and then press and hold the compass.
You can also change the map orientation in the navigation settings.

Wenn der Kompass auf automatische Ausblendung eingestellt ist, drücken Sie zuerst auf das Display, damit der Kompass angezeigt wird, und halten Sie dann den Kompass gedrückt.
Sie können die Kartenausrichtung auch in den Navigationseinstellungen ändern.

**Reply by devemux86** [reply]
@SF4User 

You can also choose to always display the compass in the map appearance settings.

Sie können in den Einstellungen zur Kartendarstellung auch festlegen, dass der Kompass immer angezeigt wird.
